test/integration/debian.py

```diff
@@ -3,7 +3,6 @@
 vm.succeed(
     "DEBIAN_FRONTEND=noninteractive sudo -E apt install /mnt/package/paretosecurity_amd64.deb -y"
 )
-
 res = vm.succeed("auditor check --json")
 fail_count = res.count("fail")
-assert fail_count == 0, f"Found {fail_count} failed checks"
+assert fail_count == 0, f"Found {fail_count} failed checkas"
```

Comment:
Is this intended?

Reply:
No. The test driver fails before we get to this, and I thought it was some sort of caching issue. In reality, it looks like the problem is with the test driver not accounting for control characters in the test output, as it uses base64 encoding to ship data from the VM to the controller.